Command/DoctrineCommand.php

```diff
@@ -32,7 +32,7 @@ public static function configureMigrations(ContainerInterface $container, Config
     {
         if (!$configuration->getMigrationsDirectory()) {
             $dir = $container->getParameter('doctrine_migrations.dir_name');
-            if (!@mkdir($dir, 0777, true) && !is_dir($dir)) {
+            if (!is_dir($dir) && !@mkdir($dir, 0777, true)) {
```

Comment:
So we have a hen-egg problem here. The current condition breaks the migration if the target directory (DoctrineMigrations) already exists. I don't know, how likely the other issue occurs, but the mentioned one is raising as soon the folder exists.